pkg/api/routes.go

```diff
@@ -60,6 +60,12 @@ func (h *Handler) GetRoutes() []Route {
 			h.CreateImage,
 		},
 		{
+			"DeleteImage",
+			"POST",
```

Comment:
Why are these methods on the RESTy API and not things requested through the operator?  I think much of the REST API functionality should go through the operator and its desired state TPR's.